src/common/components/fileUpload/utils.ts

```diff
@@ -0,0 +1,15 @@
+import { AttachmentMetadata } from '../../types/attachment';
+
+// Filter out duplicate files based on file name
+export function removeDuplicateAttachments<T extends AttachmentMetadata>(
+  files: File[],
+  attachments: T[] | undefined,
```

Comment:
Would it make sense to rename these to something like existingAttachments and newFiles?

Reply:
Good point. Renamed those.